src/main/java/at/hannibal2/skyhanni/features/misc/trevor/TrevorFeatures.kt

```diff
@@ -323,6 +324,11 @@ object TrevorFeatures {
         resetTrapper()
     }
 
+    @SubscribeEvent
+    fun onTick(event: LorenzTickEvent) {
+        inTrapperDen = LorenzUtils.skyBlockArea == "Trapper's Den"
```

Comment:
Probably update it on scoreboard change + use repopattern

Reply:
repo pattern is good.
scorboard change event is tricky: we either need to delay the LorenzUtils.skyBlockArea check by 1-2 ticks, or we need to check the scoreboard data directly. both is not ideal.

Reply:
we can do the scoreboarch change later